docs/docs/types/form.md

```diff
@@ -0,0 +1,48 @@
+Used primarily for data collection or user identification. Supporting a variety of form fields and options for passing data to various end-points this type presents a form for the user to fill out and submit.
```

Comment:
Just like in Types » Message, I think these opening sentences should mention the page name.